daemon/container_operations_unix.go

```diff
@@ -320,6 +323,10 @@ func (daemon *Daemon) buildSandboxOptions(container *container.Container, n libn
 		dns         []string
 		dnsSearch   []string
 		dnsOptions  []string
+		portSpecs   = make(nat.PortSet)
```

Comment:
looks like this can just be `portSpecs nat.PortSet` since the only thing we are doing with it is `portSpecs = container.Config.ExposedPorts` and then ranging over it.

For that matter, maybe we can get rid of the variable altogether and range over `container.Config.ExposedPorts` directly.